lms/envs/common.py

```diff
@@ -1548,6 +1549,8 @@ def _make_mako_template_dirs(settings):
 USERNAME_REGEX_PARTIAL = r'[\w .@_+-]+'
 USERNAME_PATTERN = fr'(?P<username>{USERNAME_REGEX_PARTIAL})'
 
+######## Settings for the default display behavior of course certificates ###########
+DEFAULT_CERTIFICATES_DISPLAY_BEHAVIOR = CertificatesDisplayBehaviors.END.value
```

Comment:
```suggestion
# .. setting_name: DEFAULT_CERTIFICATES_DISPLAY_BEHAVIOR
# .. setting_default: CertificatesDisplayBehaviors.END.value
# .. setting_description: Default value for the `certificates_display_behavior` course field. 
#      Together with certificate_available_date it will determine when a user can see their certificate for the course.
DEFAULT_CERTIFICATES_DISPLAY_BEHAVIOR = CertificatesDisplayBehaviors.END.value
```

Reply:
Yes, it is more informative